app/components/Table.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import $ from 'jquery'
+import Row from '../Row'
+
+class Table extends React.Component {
+
+  render() {
+    return(
+      <div>
+        <table className="table table-bordered table-rounded">
+          <tr><th>Self (ms)</th><th>Total (ms)</th><th>Method</th></tr>
+          {this.displayRows()}
+        </table>
+      </div>
+    )
+  }
+
+  displayRows() {
+    let sampleData = this.getSampleData();
+    return sampleData.forEach(sample => {
```

Comment:
It will eventually be `map`, this is just stub data. Most of this will be re-written as live data is received by this component.